test/new-e2e/tests/agent-subcommands/secret/secret_nix_test.go

```diff
@@ -63,7 +63,7 @@ host_aliases:
 	assert.Contains(v.T(), output, "Owner: dd-agent")
 	assert.Contains(v.T(), output, "Group: dd-agent")
 	assert.Regexp(v.T(), "Number of secrets .+: 1", output)
-	assert.Contains(v.T(), output, "- 'alias_secret':\n\tused in 'datadog.yaml' configuration in entry 'host_aliases'")
+	assert.Contains(v.T(), output, "- 'alias_secret':\n\tused in 'datadog.yaml' configuration in entry 'host_aliases")
```

Comment:
This feels like the code is missing a quote no ? Should we fix it rather than the test ?

Reply:
That was in order to match either `'host_aliases'` or `'host_aliases/0'`, the common prefix is `'host_aliases` (missing final quote). However I found the proper fix instead, so now it's just using `'host_aliases/0'`.